activemq-transport/src/main/java/com/esri/geoevent/transport/activemq/ActiveMQInboundTransportDefinition.java

```diff
@@ -54,10 +54,10 @@ public ActiveMQInboundTransportDefinition() {
       propertyDefinitions.put("artemisKSPath", new PropertyDefinition("ARTEMIS_KS_PATH", PropertyType.String, null, "${com.esri.geoevent.transport.activemq-transport.TRANSPORT_IN_ARTEMIS_KS_PATH_LBL}", "${com.esri.geoevent.transport.activemq-transport.TRANSPORT_IN_ARTEMIS_KS_PATH_DESC}", true, false));
       propertyDefinitions.put("artemisTSPasswd", new PropertyDefinition("ARTEMIS_TS_PASSWD", PropertyType.String, null, "${com.esri.geoevent.transport.activemq-transport.TRANSPORT_IN_ARTEMIS_TS_PASSWD_LBL}", "${com.esri.geoevent.transport.activemq-transport.TRANSPORT_IN_ARTEMIS_TS_PASSWD_DESC}", true, false));
       propertyDefinitions.put("artemisKSPasswd", new PropertyDefinition("ARTEMIS_KS_PASSWD", PropertyType.String, null, "${com.esri.geoevent.transport.activemq-transport.TRANSPORT_IN_ARTEMIS_KS_PASSWD_LBL}", "${com.esri.geoevent.transport.activemq-transport.TRANSPORT_IN_ARTEMIS_KS_PASSWD_DESC}", true, false));*/
```

Comment:
please remove this commented-out section.

Reply:
Done